Make docker-compose compatible with PyYAML < 5.3.

Upstream is not yet using PyYAML 5.1 as they are waiting for the
final 5.2 release (due api differences that does not affect compose),
so make it compatible with PyYAML < 5.3 as OE now ships with 5.1.2.

diff --git a/setup.py b/setup.py
index 043cdf9..c5bd40c 100644
--- a/setup.py
+++ b/setup.py
@@ -32,7 +32,7 @@ def find_version(*file_paths):
 install_requires = [
     'cached-property >= 1.2.0, < 2',
     'docopt >= 0.6.1, < 0.7',
-    'PyYAML >= 3.10, < 4.3',
+    'PyYAML >= 3.10, < 5.3',
     'requests >= 2.6.1, != 2.11.0, != 2.12.2, != 2.18.0, < 2.23',
     'texttable >= 0.9.0, < 0.10',
     'websocket-client >= 0.32.0, < 1.0',
